data/json/itemgroups/guns.json

```
@@ -81,13 +83,15 @@
     "id": "guns_pistol_obscure",
     "//": "Imported or otherwise very obscure pistols.",
     "items": [
+      { "item": "af2011a1_38super", "prob": 100, "charges-min": 0, "charges-max": 16 },
```

```suggestion
      { "item": "af2011a1_38super", "prob": 1, "charges-min": 0, "charges-max": 16 },
```

These things are exceptionally rare.